repo/packages/G/geoserver/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion" : "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name" : "geoserver",
+  "version" : "2.10.0-1.1.0",
+  "scm" : "https://github.com/appliedis/dcos-geoserver",
+  "maintainer" : "dcos-support@appliedis.com",
+  "description" : "GeoServer 2.10.0 web application server implementating OGC Web Map Service (WMS), Web Feature Service (WFS), Web Coverage Service (WCS) and Catalog Serivce for the Web (CSW).",
```

Comment:
Please explain what `geoserver` is, OGC WMS...? link them to a documentation site if one exists.